UpdateRCConfiguration/__tests__/updateRCConfiguration.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+import * as O from "fp-ts/lib/Option";
+
+import {
+  handleEmptyConfiguration,
+  handleGetLastRCConfigurationVersion,
+  handleUpsert,
+  isUserAllowedToUpdateConfiguration
+} from "../handler";
+import {
+  aRemoteContentConfiguration,
+  findLastVersionMock,
+  rccModelMock,
+  upsertConfigurationMock
+} from "../../__mocks__/remote-content";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+describe("isUserAllowedToUpdateConfiguration", () => {
+  test("should return a left if the userId is not equal to the userId of the configuration", async () => {
+    const r = await isUserAllowedToUpdateConfiguration(
+      "aDifferentUserId" as NonEmptyString
+    )(aRemoteContentConfiguration)();
+    expect(E.isLeft(r)).toBeTruthy();
+  });
+
+  test("should return a right if the userId is equal to the userId of the configuration", async () => {
+    const r = await isUserAllowedToUpdateConfiguration(
+      aRemoteContentConfiguration.userId
+    )(aRemoteContentConfiguration)();
+    expect(E.isRight(r)).toBeTruthy();
```

Comment:
ditto.
We should use truthy and falsy when we expect a range of values. Here we know that the value must be true and only true.